### ISSUE TITLE
schema-reader Error: Undefined array key "Name"

### ISSUE THREAD

**Issue by grafst** [VemtoOrg/vemto2-issues#53]
**Error Message:**
schema-reader Error: Undefined array key "Name"

**Stack:**
```
schema-reader Error: #0 /Users/stephan/git/web3/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(255): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Undefined array...', 'phar:///Applica...', 143)VEMTO_EOL#1 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(143): Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(2, 'Undefined array...', 'phar:///Applica...', 143)VEMTO_EOL#2 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(129): TableRepository->renameTableColumn(Array)VEMTO_EOL#3 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(24): TableRepository->processRenamedColumns(Array)VEMTO_EOL#4 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/index.php(82): TableRepository->buildTablesFromMigrations()VEMTO_EOL#5 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/common/Vemto.php(107): {closure}()VEMTO_EOL#6 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/index.php(18): Vemto::execute('schema-reader', Object(Closure))VEMTO_EOL#7 /Applications/Vemto.app/Contents/Resources/static/VMTTL1(15): require('phar:///Applica...')VEMTO_EOL#8 {main}
```

---

Vemto 2.0.6
darwin 23.4.0
Locale: de
App version: 2.0.6

**Comment by TiagoSilvaPereira** [comment]
@grafst is this a new project or an existing one?

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.7
